packages/dnb-eufemia/src/components/skeleton/SkeletonHelper.d.ts

```diff
@@ -23,8 +23,10 @@ export interface SkeletonContextProps {
   };
 }
 
+export type SkeletonMethodes = 'shape' | 'font' | 'code';
```

Comment:
```suggestion
export type SkeletonMethods = 'shape' | 'font' | 'code';
```

I think `methodes` could work as well, but it is an ancient way of spelling it 👴

Reply:
Sorry for the confution! The `inner_ref` is an very old thing. Later I needed `internalClass` and because we prefer now camelCase (and Web Components support is not needed anymore) it mate it in as camelCase.

Anyway, I think, also the Anchor, and all other re-written TS components (Slider and Modal) an get an additional internal re-write, where we rather rewrite them to camelCase, but provide a `withSnakeCaseProps` – instead of `withCamelCaseProps` as we do now.